aspnetcore/fundamentals/minimal-apis/openapi.md

```diff
@@ -182,7 +182,7 @@ app
     .Produces<IList<Todo>>();
 ```
 
-Leveraging [`TypedResults`](/dotnet/api/microsoft.aspnetcore.http.typedresults) in the implementation of an endpoint's route handler automatically includes the response type metadata for the endpoint. For example, the code below automatically annotates the endpoint with a response under the `200` status code with an `application/json` content type.
+Using [`TypedResults`](/dotnet/api/microsoft.aspnetcore.http.typedresults) in the implementation of an endpoint's route handler automatically includes the response type metadata for the endpoint. For example, the following code automatically annotates the endpoint with a response under the `200` status code with an `application/json` content type.
```

Comment:
I frequently make this same change Leveraging -> Using. I'm guessing Leveraging is used frequently in their planning.